jdaviz/configs/default/plugins/line_lists/line_lists.vue

```diff
@@ -248,6 +248,12 @@
                 ></v-text-field>
               </v-row>
 
+              <v-row class="row-no-vertical-padding-margin" style="margin: 0px" justify="end">
+                <span class='text--primary' style="overflow-wrap: anywhere; font-size: 8pt;">
+                  Medium: {{list_contents[item].medium}}
+                </span>                    
+              </v-row>
```

Comment:
is it better to display "Unknown" or skip displaying altogether in those cases?

Reply:
I figured it would be better to indicate explicitly to the user so that they're aware. The scenario I'm trying to avoid is the user loads a line list that doesn't have medium info, and just blanket assumes it's one of them because of their field of study. This way, it's almost like an explicit warning, but gentler